example/App.js

```diff
@@ -1,327 +1,498 @@
-      },
-    ],
-    mode: 'car',
-    units: 'imperial',
-  }).then((result) => {
-    console.log('getMatrix:', stringify(result));
-  }).catch((err) => {
-    console.log('getMatrix:', err);
-  });
+  const stringify = (obj) => JSON.stringify(obj, null, 2);
 
-  /*Radar.startTrip({
-    externalId: '299',
-    destinationGeofenceTag: 'store',
-    destinationGeofenceExternalId: '123',
-    mode: 'car',
-    scheduledArrivalAt: new Date('2023-10-10T12:20:30Z').getTime()
-  });*/
+  useEffect(() => {
+    Radar.initialize("prj_test_pk_4ce462a534c0f749af3fdaac6fff9f485d65e720");
 
-  /*Radar.startTrip({
-    tripOptions: {
-      externalId: '300',
-      destinationGeofenceTag: 'store',
-      destinationGeofenceExternalId: '123',
-      mode: 'car',
-      scheduledArrivalAt: new Date('2023-10-10T12:20:30Z').getTime()
-    }
-  }).then((result) => {
-    console.log('startTrip:', stringify(result));
-  }).catch((err) => {
-    console.log(err.stack);
-  });*/
+    Radar.setLogLevel("info");
 
-  Radar.startTrip({
-    tripOptions: {
-      externalId: '302',
-      destinationGeofenceTag: 'store',
-      destinationGeofenceExternalId: '123',
-      mode: 'car',
-      scheduledArrivalAt: new Date('2023-10-10T12:20:30Z').getTime()
-    },
-    trackingOptions: {
-      desiredStoppedUpdateInterval: 30,
-      fastestStoppedUpdateInterval: 30,
-      desiredMovingUpdateInterval: 30,
-      fastestMovingUpdateInterval: 30,
-      desiredSyncInterval: 20,
-      desiredAccuracy: "high",
-      stopDuration: 0,
-      stopDistance: 0,
-      replay: "none",
-      sync: "all",
-      showBlueBar: true,
-      useStoppedGeofence: false,
-      stoppedGeofenceRadius: 0,
-      useMovingGeofence: false,
-      movingGeofenceRadius: 0,
-      syncGeofences: false,
-      syncGeofencesLimit: 0,
-      beacons: false,
-      foregroundServiceEnabled: false
-    }
-  }).then((result) => {
-    console.log('startTrip:', stringify(result));
-  }).catch((err) => {
-    console.log('startTrip:', err);
-  });
+    Radar.setUserId("foo");
 
-  Radar.mockTracking({
-    origin: {
-      latitude: 40.78382,
-      longitude: -73.97536,
-    },
-    destination: {
-      latitude: 40.70390,
-      longitude: -73.98670,
-    },
-    mode: 'car',
-    steps: 3,
-    interval: 3,
-  });
+    Radar.setDescription("description");
 
-  Radar.completeTrip();
-  
-  Radar.sendEvent(
-    "in_app_purchase",
-    {"price": "$150"}
-  ).then((result) => {
-    console.log("sendEvent:", stringify(result));
-  }).catch((err) => {
-    console.log("sendEvent:", err);
-  });
+    Radar.setMetadata({
+      foo: "bar",
+      baz: true,
+      qux: 1,
+    });
+  }, []);
 
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
+      <View style={{ width: "100%", height: "50%" }}>
+        <Map />
+      </View>
+      <View style={{ width: "100%", height: "15%" }}>
```

Comment:
I'd suggest making the heights: 40%, 10% and then 50% for the next -- think that looks the best.